.github/workflows/linkcheck.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       - name: Test HTML
         # https://github.com/wjdp/htmltest-action/
         # Don't fail the build on broken links
-        continue-on-error: false
+        continue-on-error: true
```

Comment:
We were getting CI failures due to a broken link; given the comment above, I think this should be `true`.